qa/tasks/vstart_runner.py

```diff
@@ -325,9 +325,14 @@ def _perform_checks_and_return_list_of_args(self, args, omit_sudo):
 
         return args
 
-    def run(self, args, check_status=True, wait=True,
-            stdout=None, stderr=None, cwd=None, stdin=None,
-            logger=None, label=None, env=None, timeout=None, omit_sudo=False):
+    # Wrapper to keep the interface exactly same as that of
+    # teuthology.remote.run.
+    def run(self, **kwargs):
+        return self.cmdrunner(**kwargs)
+
+    def cmdrunner(self, args, check_status=True, wait=True, stdout=None,
```

Comment:
could you prefix the method with "_" to note that this is an internal method? and rename it to `_do_run()`. as `cmdrunner` is not a member variable?

Reply:
Fixed, thanks!